### ISSUE TITLE
undefined variable 'emulator'

### ISSUE THREAD

**Issue by TonyGiorgio** [damus-io/notedeck#46]
I'm on `aarch64-linux` and `nix (Nix) 2.13.5` and can't find the emulator package.

```
notedeck git:(master) nix-shell                                                                                                                                                                                                                   
error: undefined variable 'emulator'                                                                                                                                                                                                                 
                                                                                                                                                                                                                                                     
       at shell.nix:16:5:                                                                                                                                                                                      
                                                                                                                                                                                                                                                     
           15|     platforms-android-30                                                                                                                                                                                                              
           16|     emulator                                                                                                                                                                                                                          
             |     ^                                                                                                                                                                                                                                 
           17|     ndk-24-0-8215888                                                                                                                                                                                                                  
(use '--show-trace' to show detailed location information)  
```

It worked after I removed this line.

**Comment by jb55** [comment]
On Fri, May 03, 2024 at 08:57:25PM GMT, Tony Giorgio wrote:
>I'm on `aarch64-linux` and `nix (Nix) 2.13.5` and can't find the emulator package.
>
>```
>notedeck git:(master) nix-shell
>error: undefined variable 'emulator'
>
>       at shell.nix:16:5:
>
>           15|     platforms-android-30
>           16|     emulator
>             |     ^
>           17|     ndk-24-0-8215888
>(use '--show-trace' to show detailed location information)
>```
>
>It worked after I removed this line.

hmm ok I'm not sure why that's there. maybe @kernelkind uses it. We can
remove it for that arch.

**Comment by jb55** [comment]
should be fixed in 30e9aa5357d2e220a8a5b57d5617a0426443cd5e

thanks for the report!
